### ISSUE TITLE
Issue With SQLite

### ISSUE THREAD

**Issue by prabhnoor10** [discordjs/guide#82]
I get his error:

`(node:1848) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): SequelizeDatabaseError: SQLITE_ERROR: no such table: users`

Here is what else I get:

`sequelize deprecated String based operators are now deprecated. Please use Symbol based operators for better security, read more at http://docs.sequelizejs.com/manual/tutorial/querying.html#operators ..\..\..\node_modules\sequelize\lib\sequelize.js:242:13`
`(node:1848) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.`

**Comment by lloti** [comment]
Did you `sequelize.sync()`?

Also, add `operatorsAliases: false` to the options when doing `new Sequelize` to get rid of the deprecation warning.

**Comment by prabhnoor10** [comment]
Yup have `sequelize.sync()` in there. All the happens atm, is that a database file is created but the bot does not launch. I've checked `https://discordapp.com/developers/applications/me` to confirm I have the right info.

**Comment by prabhnoor10** [comment]
Also this is only happening with the currency, I started the bot fine with sequelize.js and other applications.
